.github/workflows/versioning.yml

```diff
@@ -41,8 +41,11 @@ jobs:
           cache-dependency-path: '**/yarn.lock'
 
       - name: Update version
-        run: yarn version --new-version ${{ env.PACKAGE_VERSION }} --no-git-tag-version
+        run: npm version ${{ env.PACKAGE_VERSION }} --no-git-tag-version --workspaces-update=false
 
+      - name: Install NPM dependencies
+        run: yarn install --immutable
```

Comment:
this is required to run any yarn command. applying the same changes made to main.yml gh action: https://github.com/coinbase/coinbase-wallet-sdk/commit/c7442551178e4a19254258796a8c791f6d6f8d11